src/test/regress/sql/create_view.sql

```diff
@@ -616,10 +616,34 @@ select pg_get_ruledef(oid, true) from pg_rewrite
   where ev_class = 'tt23v'::regclass and ev_type = '1';
 
 
+<<<<<<< HEAD
```

Comment:
I will resolve conflicts in the `create_view.sql`

Reply:
It has conflicts in the output too, should you edit it without checking?

Reply:
I was going to fix only the `sql` part. Do you think it's worth trying to fix `out`?

Reply:
I think it will be easier to fix conflicts in both files at once after checking, which can be done immediately after compilation.

Reply:
Although now you can fix the `sql`, and after compilation and testing add fixes for `out` to the same patch.

Reply:
https://github.com/arenadata/gpdb/pull/1326